src.es5.js

```diff
@@ -4,6 +4,25 @@ var config = {
 	divider: '\n---'
 };
 
+function onLinkedInLoad() {
+	IN.Event.on(IN, "auth", getProfileData);
+}
+
+// Handle the successful return from the API call
+function onSuccess(data) {
```

Comment:
the `on*` naming for these functions are too generic to be in global scope. We'd need to move these into a closure or as object methods